raiden_contracts/deploy/__main__.py

```diff
@@ -87,6 +95,7 @@ def wrapper(*args: List, **kwargs: Dict) -> Any:
 def setup_ctx(
     ctx: click.Context,
     private_key: str,
+    password_file: Optional[Path],
```

Comment:
Would that ever work if one doesn't provide a password file? Or, why is this optional?

Reply:
If not password file is given, you get a password prompt.